src/main/java/core/basesyntax/reader/CsvFileReader.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.reader;
+
+import com.opencsv.CSVParser;
+import com.opencsv.CSVParserBuilder;
+import com.opencsv.CSVReader;
+import com.opencsv.CSVReaderBuilder;
+import com.opencsv.exceptions.CsvValidationException;
+import core.basesyntax.exceptions.FileWasNotReadException;
+import core.basesyntax.handler.TransactionHandler;
+import core.basesyntax.handler.impl.BalanceTransactionHandler;
+import core.basesyntax.handler.impl.PurchaseTransactionHandler;
+import core.basesyntax.handler.impl.ReturnTransactionHandler;
+import core.basesyntax.handler.impl.SupplyTransactionHandler;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CsvFileReader {
+    private final String filePath;
+    private final Map<String, TransactionHandler> transactionHandlers = new HashMap<>();
```

Comment:
remove those fields
filePath can be a local variable for method that loads the file
handlers being here violate SRP